packages/react/carbon.yml

```diff
@@ -532,6 +561,20 @@ assets:
         url: https://react.carbondesignsystem.com/?path=/docs/experimental-unstable-text
     tags:
       - input-control
+  text-direction:
+    name: Unstable text direction
+    framework: react
+    status: experimental
+    type: component
+    platform: web
+    thumbnailPath: ./thumbnails/text.svg
```

Comment:
Do we want the same thumbnail as Text? Or should we remove `thumbnailPath`? @jeanservaas